virtualenv.py

```diff
@@ -1191,6 +1191,9 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
     site_filename_dst = change_prefix(site_filename, home_dir)
     site_dir = os.path.dirname(site_filename_dst)
     writefile(site_filename_dst, SITE_PY)
+    if is_win:
+        fixtk_filename = join(site_dir, 'FixTk.py')
+        writefile(fixtk_filename, FIXTK_PY)    
```

Comment:
> Hmm, wait. I see there's a Lib\lib-tk\FixTk.py in Python 2.7. So this fix doesn't work on Python 3 (where there's no lib-tk directory). Can it be updated to cover that as well?

The fix file also exist in Python 3, here: "C:\Python34\Lib\tkinter\_fix.py"
it's simmilar content to "C:\Python27\Lib\lib-tk\FixTk.py"